torch/ao/sparsity/experimental/pruner/base_pruner.py

```diff
@@ -44,45 +51,170 @@ def _prepare(self, use_path=False, *args, **kwargs):
         self.bias_handles = []
 
         for config in self.module_groups:
+            modules = []
             if use_path:
-                module = fqn_to_module(self.model, config['fqn'])
+                if type(config['module']) is tuple:  # (Conv2d, BN)
```

Comment:
Would be great if the `self.module_groups` variable was documented in the class docblock.